lib/glimesh/graphql/schema.ex

```diff
@@ -4,6 +4,7 @@ defmodule Glimesh.Schema do
   """
 
   use Absinthe.Schema
+  use Absinthe.Relay.Schema, :modern
```

Comment:
Is this needed? I think we may only need it on the new lib/glimesh/api/schema.ex

Reply:
Might have been something I forgot to remove once I moved it to its own schema for relay and old